tools/backups2datalad-cron-common

```diff
@@ -14,7 +14,8 @@ conda activate dandisets-2
 cd "$ds"
 chronic pip install -r tools/backups2datalad.req.txt
 
-export DATALAD_LOG_LEVEL=WARNING  # otherwise too much noise
+: "${DATALAD_LOG_LEVEL:=WARNING}"  # otherwise too much noise
+export DATALAD_LOG_LEVEL
```

Comment:
any particular reason on why not in one line?

```suggestion
export "${DATALAD_LOG_LEVEL:=WARNING}"  # otherwise too much noise
```

Reply:
ha, interesting... if to be formal indeed it should be that way.  I just did empirically in the shell and it mysteriously worked so I thought that it is ok. 

```
*$> export ${BUGA:=duga}

$> echo $BUGA
duga

*$> export ${BUGA2:=123}
bash: export: `123': not a valid identifier
```

so it is all odd , let's do your clean way. Thank you!